### PR TITLE
Add langtool style and grammar checker for 25 lang

### PR DESCRIPTION
**Pull request by wigust**
LanguageTool is an Open Source proofreading software for English, French,
German, Polish, Russian, and more than 20 other languages. It finds many errors
that a simple spell checker cannot detect.

For more information, please see homepage at https://languagetool.org

### CODE REVIEW

File: layers/+checkers/langtool/README.org [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+** Java
+
+Install =openjdk-8-jdk= or =openjdk-8-jre= and set default java to version 8.
+
+Debian/Ubuntu:
+#+BEGIN_EXAMPLE
+sudo apt-get install openjdk-8-jdk
+sudo update-alternatives --config java
+#+END_EXAMPLE
```
Comment (LemmingAvalanche) [inline layers/+checkers/langtool/README.org]: This could be `BEGIN_SRC shell` instead.

File: layers/+checkers/langtool/packages.el [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+(defconst langtool-packages
+  '(langtool))
+
+;; Author: Steve Shogren https://github.com/steveshogren
+(defun langtool/init-langtool ()
+    (use-package langtool
+        :defer t
+        :init
+        (progn
+          (global-set-key "\M-n" 'langtool-goto-next-error)
```
Comment (LemmingAvalanche) [inline layers/+checkers/langtool/packages.el]: What’s the motivation for choosing this keybinding?

Comment (wigust) [inline layers/+checkers/langtool/packages.el]: No motivation. It was free for binding in holy mode. I think need to change it
to something else according to Spacemacs key binds style.

**Comment by nixmaniack** [comment]
Reminds me of https://github.com/syl20bnr/spacemacs/issues/5996#issuecomment-219238127.

**Comment by LemmingAvalanche** [comment]
This kind of tooling is exactly what I want to see more of in things like power-editors. Thank you for your efforts.

**Comment by samwindels** [comment]
Any word on when/if this is coming to develop?

**Comment by d12frosted** [comment]
@samwindels For some reason it was closed by OP. 

Usually new layers are merged directly by Sylvain. He is a little bit busy with some other stuff right now, so new layers are not being merged as fast as some users/contributors would like, but don't worry, in case anything is needed - it will be inside sooner or later 😸

**Comment by samwindels** [comment]
Thanks for the quick reply! I also stumbled on the [proselint  checker](https://github.com/flycheck/flycheck/pull/939) . Exciting stuff!

**Comment by d12frosted** [comment]
@samwindels nice. Proselint support is added in [this PR][1].

  [1]: https://github.com/syl20bnr/spacemacs/pull/6012

**Pull request closed without merging**